fe/fe-core/src/main/java/org/apache/doris/backup/Repository.java

```diff
@@ -102,6 +102,7 @@ public class Repository implements Writable {
     private static final Logger LOG = LogManager.getLogger(Repository.class);
     private static final String PATH_DELIMITER = "/";
     private static final String CHECKSUM_SEPARATOR = ".";
+    private static final String PROPERTY_DELETE_STALED_SNAPSHOTS = "s3.delete_staled_snapshots";
```

Comment:
I think that it is not only for s3 repo, but should for all kinds of repo (hdfs/broker)
And how about name it `delete_if_exists`?